### DISCUSSION TITLE
[Bug] Claude 3 中文乱码

### DISCUSSION THREAD

**Discussion by chain710** [lobehub/lobehub#1804] [General Question] [answered]
### 💻 Operating System

Other Linux

### 📦 Environment

Docker

### 🌐 Browser

Chrome

### 🐛 Bug Description

![image](https://github.com/lobehub/lobe-chat/assets/486539/5b5afc89-0144-4f7e-a372-58b71248b6b7)

### 🚦 Expected Behavior

无乱码

### 📷 Recurrence Steps

偶发出现，无重现步骤

### 📝 Additional Information

使用的AI服务是：Claude 3 Sonnet

[a comment by an automated account is not shown]

**Comment by CUGbaoyi** [comment, marked as answer]
这应该是Claude自己的问题，我在用它官方网页版的时候偶尔也会出现这个问题

**Reply by MapleEve** [reply]
对的，接口返回的也是乱码，Kimi 也有这样的问题

[a comment by an automated account is not shown]
